src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -259,17 +268,16 @@ const ModalCardDetails = ({
     type: ModalPaymentTypes.modalPages,
     amount: number
   ) => {
-    if (sellerId === 'send-chinatown-love')
-      type = ModalPaymentTypes.modalPages.donation_pool;
-
-    if (
-      type === ModalPaymentTypes.modalPages.gift_card ||
-      (type === ModalPaymentTypes.modalPages.light_up_chinatown &&
-        amount >= LIGHT_UP_CHINATOWN_TIER_2_MIN)
-    ) {
-      return t('modalPayment.modalCardDetails.details.voucher');
-    } else {
-      return t('modalPayment.modalCardDetails.details.donation');
+    switch (true) {
```

Comment:
probably worth keeping the if statement syntax if we're not actually switching on anything here

Reply:
unrelated but I was surprised we could use bool in a switch case

Reply:
no problem.i can convert it to if statements. I was trying to simplify the logic on the hand side but can easily switch the case statement to if statements